modules/ipi-install-network-requirements.adoc

```diff
@@ -154,4 +154,4 @@ You can reconfigure the control plane nodes to act as NTP servers on disconnecte
 [id='network-requirements-out-of-band_{context}']
 == Port access for the out-of-band management IP address
 
-The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the `baremetal` node during installation, the out-of-band management IP address must be granted access to port `6180`.
\ No newline at end of file
+The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the `baremetal` node during installation, the out-of-band management IP address must be granted access to port `6180`.
```

Comment:
What is the 'baremetal' node? 
I'm not familiar with this prerequisite so I may be missing something, but I never hear of a .baremetal node'.
Also, for some reason, in the document itself, I see the work 'address' twice.